src/lab11-tests.md

```diff
@@ -154,7 +157,7 @@ destination-in compositing yields the backdrop.
 But transparency multiplies.
 
     >>> red_opaque.copy().destination_in(blue_semitransparent)
-    Pixel(0.5, 0.0, 0.0, 0.5)
+    Pixel(1.0, 0.0, 0.0, 0.5)
```

Comment:
And my comment above leads me to conclude the red channel should be 0.5.

Reply:
How does that make sense? Destination in blending doesn’t make the color darker!

Reply:
I guess it does if the destination has opacity?

We could test this with a page that uses opacity on the white rounded clipping rectangle and see what it does in SKia.

Reply:
Ok, so here's my test Skia program: https://fiddle.skia.org/c/2da37d5e990775229a208b41bdace46f

It draws an opaque red rectangle and then draws a semitransparent blue circle over it with DstIn. The resulting overlapped pixels are `rgb(1, 0, 0, .5)`. So I think the PR is correct, the original test was wrong.